docs/layers.md

```diff
@@ -2108,6 +2108,80 @@ Transit lines may have their colours mapped onto one of these CSS colours. The i
 * `yellow`
 * `yellowgreen`
 
+## Traffic Flow Layer
+This **optional** layer is meant to be updated every few minutes, due to the highly dynamic nature of the data. Consideration should be given to rendering the features from this layer linked to the **roads** layer when linear referencing is available, or as an overlay
+
+* Layer name: `traffic_flow`
+* Geometry types: `line`
+
+#### `common` properties:
+* `id`: Unique traffic event ID. Can be referenced when checking for updated traffic information for specified event
+* `kind`: the **severity** of the flow information as seen by traffic provider to indicate the traffic "color"
+* `kind_detail`: the **congestion** level of traffic flow, with 0.0 representing completely free flowing traffic to 1.0 completely congested traffic
```

Comment:
Let's move `congression` down to `common-optional` properties section and call it `congestion` not `kind_detail`, to go along with `speed` down there.